test/integration/patients/patient/patient-flow.js

```diff
@@ -1,65 +1,71 @@
 import _ from 'underscore';
+import { v4 as uuid } from 'uuid';
 
 import { testTs, testTsSubtract } from 'helpers/test-timestamp';
 import { testDateAdd, testDateSubtract } from 'helpers/test-date';
-import { getErrors } from 'helpers/json-api';
+import { getErrors, getRelationship, mergeJsonApi } from 'helpers/json-api';
+
+import { getFlow } from 'support/api/flows';
+import { getPatient } from 'support/api/patients';
+import { getAction } from 'support/api/actions';
+import { getProgramAction } from 'support/api/program-actions';
+import { getProgramFlow } from 'support/api/program-flows';
+import { getClinician, getCurrentClinician } from 'support/api/clinicians';
+import { getForm, testForm } from 'support/api/forms';
+import { stateInProgress, stateDone, stateTodo } from 'support/api/states';
+import { teamCoordinator, teamNurse, teamPharmacist, teamPhysician } from 'support/api/teams';
+import { roleNoFilterEmployee, roleTeamEmployee } from 'support/api/roles';
 
 const tomorrow = testDateAdd(1);
 
 context('patient flow page', function() {
+  const testFlowId = uuid();
+
+  const testFlow = getFlow({
+    id: testFlowId,
```

Comment:
you don't need the `testFlowId` here you can use `testFlow.id` later and just use `getRelationship(testFlow)` below.. you would need a `uuid` if the flow also related to the action

Reply:
Updated 👍 And switched to `getRelationship(testFlow)` throughout.